course_discovery/apps/api/v1/views.py

```
@@ -280,13 +307,21 @@ def contains(self, request):
               type: string
               paramType: query
               multiple: true
+            - name: partner
+              description: Filter by partner
+              required: false
+              type: string
+              paramType: query
+              multiple: false
         """
         query = request.GET.get('query')
         course_run_ids = request.GET.get('course_run_ids')
+        partner_name = self._get_partner_name_from_code()
 
-        if query and course_run_ids:
+        if query and course_run_ids and partner_name:
```

Do we want to raise the InvalidPartnerError here as well?